contentcuration/contentcuration/frontend/shared/views/ActionLink.vue

```diff
@@ -3,14 +3,20 @@
   <VBtn
     :style="{ color: $vuetify.theme.primary }"
     :target="target"
+    :class="{ truncate }"
     v-bind="$attrs"
     flat
     exact
     @keyup.enter.stop="$emit('click')"
     @click.stop="$emit('click')"
   >
     <slot>
-      <span style="text-decoration: underline;">
+      <span
+        :class="{ 'text-truncate': truncate }"
+        :title="text"
+        class="notranslate"
```

Comment:
We might want to make this a prop as well in case there are action links that need to be translated (e.g. the "Frequently asked questions" link on the catalog)

Reply:
Oh good catch. I'll update it